app/segments/drill/[id]/submission/input.js

```diff
@@ -160,195 +333,178 @@ export default function Input({ inputValues, setInputValues }) {
     <GestureHandlerRootView style={{ flex: 1 }}>
       <PaperProvider>
         <SafeAreaView>
-          <BottomSheetModalProvider>
-            <Appbar.Header
-              style={{ backgroundColor: "FFF" }}
-              statusBarHeight={0}
-            >
-              <Appbar.BackAction
-                onPress={showLeaveDrillDialog}
-                color={"#F24E1E"}
-              />
-              <Appbar.Content
-                title="20 Shot Challenge"
-                titleStyle={styles.title}
-              />
-              <Appbar.Action
-                icon="information-outline"
-                onPress={() => {
-                  handlePresentDesciptionModalPress();
-                }}
-                color={"#F24E1E"}
-              />
-            </Appbar.Header>
-            {/* Empty Input Banner */}
-
-            <Banner
-              visible={emptyInputBannerVisable}
-              actions={[
-                {
-                  label: "Dismiss",
-                  onPress: () => setEmptyInputBannerVisable(false),
-                },
-              ]}
-            >
-              Error! All input fields must be filled!
-            </Banner>
-
-            <KeyboardAwareScrollView>
-              {/* Shot Number / Total shots */}
-              <View style={styles.shotNumContainer}>
-                <Text style={styles.shotNumber}>
-                  Shot {AttemptData.shots[shotIndex].shotNum}
-                  <Text style={styles.shotTotal}>
-                    /{AttemptData.shots.length}
+          <View style={{ height: "100%" }}>
+            <BottomSheetModalProvider>
+              <Appbar.Header
+                style={{ backgroundColor: "FFF" }}
+                statusBarHeight={0}
+              >
+                <Appbar.Action
+                  icon="close"
+                  onPress={showLeaveDrillDialog}
+                  color={"#F24E1E"}
+                />
+                <Appbar.Content title={drillTitle} titleStyle={styles.title} />
+                <Appbar.Action
+                  icon="information-outline"
+                  onPress={() => {
+                    handlePresentDesciptionModalPress();
+                  }}
+                  color={"#F24E1E"}
+                />
+              </Appbar.Header>
+              {/* Empty Input Banner */}
+
+              <Banner
+                visible={emptyInputBannerVisable}
+                actions={[
+                  {
+                    label: "Dismiss",
+                    onPress: () => setEmptyInputBannerVisable(false),
+                  },
+                ]}
+              >
+                Error! All input fields must be filled!
+              </Banner>
+
+              <KeyboardAwareScrollView>
```

Comment:
Bit of a nitpick, but there are inconsistencies with how the input screen looks while typing on iOS vs Android. The iOS version (at least for me) doesn't show the "Next Shot" button until you close the keyboard. 

This can probably be solved by wrapping the "Next Shot" and "View all Shots" components inside the KeyboardAwareScrollView, to ensure (relatively) consistent scrolling / keyboard behavior between iOS and Anddroid.

Android
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/82061589/df38380a-4169-424e-9144-1cc64d88aa0e)

iOS
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/82061589/18f271ac-e7d4-4676-8e6d-13f885e40aab)

Reply:
I did what you suggested and it gives a relatively similar result on both android and ios